skcosmo/datasets/_base.py

```diff
@@ -0,0 +1,32 @@
+from os.path import join, dirname
+import numpy as np
+from sklearn.utils import Bunch
+
+
+def load_degenerate_manifold():
+    """Load and return the degenerate manifold dataset.
+
+    Returns
+    -------
+    degenerate_manifold_data : sklearn.utils.Bunch
+        Dictionary-like object, with the following attributes:
+
+        data : `sklearn.utils.Bunch` --
+        contains the keys ``SOAP_power_spectrum`` and ``SOAP_bispectrum``.
+        Two representations of the carbon environments of the
+        degenerate manifold dataset.
+
+        DESCR: `str` --
+        The full description of the dataset.
+    """
+    module_path = dirname(__file__)
+    target_filename = join(module_path, "data", "degenerate_manifold.npz")
+    raw_data = np.load(target_filename)
+    data = Bunch(
+        SOAP_power_spectrum=raw_data["SOAP_power_spectrum"],
+        SOAP_bispectrum=raw_data["SOAP_bispectrum"],
+    )
+    with open(join(module_path, "descr", "degenerate_manifold.rst")) as rst_file:
+        fdescr = rst_file.read()
+
```

Comment:
Are you choosing not to give the option to return as pandas dataframe?

Reply:
Why would we want to use pandas? I don't think it is a dependency of sklearn, so I don't see why we would want to bring it in.

Reply:
> This is an available functionality in sklearn.datasets, that’s why I brought it up

I can't find anything related to pandas in https://scikit-learn.org/stable/datasets/index.html, except for the section on "loading your own dataset". Where is this documented?

Reply:
There argument `as_frame` offers panda support, e.g.
https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_iris.html#sklearn.datasets.load_iris

I chose to not do it to simplify the process. Can I make it an issue and do this for another PR?

Reply:
I think it requires to copy some sklearn internal checkers, if done properly, and is not only one line to add.

Reply:
Agree that this can be done later if we want. Seems that not all sklearn dataset offer this: https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html#sklearn.datasets.load_boston

Reply:
opened issue #19